### PR TITLE
Create README.md

### PR DESCRIPTION
**Pull request by rdenadai**
README.md for my model

https://huggingface.co/rdenadai/BR_BERTo

### CODE REVIEW

File: model_cards/rdenadai/BR_BERTo/README.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
+---
+language: "UTF-8"
```
Comment (julien-c) [inline model_cards/rdenadai/BR_BERTo/README.md:2]:
```suggestion
language: pt
```

File: model_cards/rdenadai/BR_BERTo/README.md [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+tags:
+- pytorch
+- roberta
+- lm-head
+- masked-lm
```
Comment (julien-c) [inline model_cards/rdenadai/BR_BERTo/README.md:7]:
```suggestion
```

Those should be automatic

**Review by rdenadai — COMMENTED** [review]
Agreed with the changes

File: model_cards/rdenadai/BR_BERTo/README.md [hunk 3]
```diff
@@ -0,0 +1,54 @@
+---
+language: "UTF-8"
```
Comment (julien-c) [inline model_cards/rdenadai/BR_BERTo/README.md:2]:
```suggestion
language: pt
```

**Comment by julien-c** [comment]
👍 

Would you like to add example inputs for all models for `pt`? If you do, please open a PR against this file: https://github.com/huggingface/widgets-server/blob/master/DefaultWidget.ts

Thanks:)

**Pull request merged**